src/tokens/solana.tokenlist.json

```diff
@@ -103705,10 +103705,10 @@
       "symbol": "HERD",
       "name": "Herd Token",
       "decimals": 9,
-      "logoURI": "https://raw.githubusercontent.com/TheChillElephantsClub/assets/main/HERD_COIN_LOGO.png",
+      "logoURI": "https://raw.githubusercontent.com/TheChillElephants/assets/main/%24HERD%20token%20logo.png",
       "tags": [
         "social-token",
-        "nft"
```

Comment:
Simple replacement of token tag, mistakenly tagged as "nft" upon initial token creation.